### DISCUSSION TITLE
Change content of hover-over popup

### DISCUSSION THREAD

**Discussion by ve3meo** [pawelsalawa/sqlitestudio#4834] [Ideas]
Currently, hovering over a cell in query results pops out the statement that produced the result. It would be more useful if would pop out the content, especially when the field cannot display the entire content. Perhaps some prefer to see the formula so it could be a viewing option.
![image](https://github.com/pawelsalawa/sqlitestudio/assets/38633449/90df3312-acb4-410f-bc0b-b6231de07093)

**Comment by pawelsalawa** [comment]
Created #4836 to track and implement it.
